app/src/marketplace/containers/ProductPage/Streams.jsx

```diff
@@ -1,30 +1,31 @@
 // @flow
 
-import React, { useCallback, useMemo } from 'react'
+import React, { useCallback, useMemo, useEffect, useState } from 'react'
 import { useSelector } from 'react-redux'
-import { withRouter } from 'react-router-dom'
+import { useHistory } from 'react-router-dom'
 
 import Segment from '$shared/components/Segment'
 import { useController } from '$mp/containers/ProductController'
 import useProductSubscription from '$mp/containers/ProductController/useProductSubscription'
-import {
-    selectStreams,
-    selectFetchingStreams,
-} from '$mp/modules/product/selectors'
 import { selectUserData } from '$shared/modules/user/selectors'
+import { usePending } from '$shared/hooks/usePending'
 import StreamListing from '$mp/components/ProductPage/StreamListing'
 import { isPaidProduct } from '$mp/utils/product'
 import routes from '$routes'
 
-const Streams = withRouter(({ history }) => {
-    const { product } = useController()
+const PAGE_SIZE = 5
+const INITIAL_OFFSET = 2 * PAGE_SIZE
```

Comment:
These defaults might need fixing, now showing 10 by default and 5 on loading more.